<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminScreenUI</class>
 <widget class="QDialog" name="AdminScreenUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="ordersList">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearOrderButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
